packages/nextra/src/components/pre.tsx

```diff
@@ -42,17 +43,20 @@ export const Pre = ({
       </pre>
       <div
         className={[
-          'nextra-code-block-buttons opacity-0 transition-opacity [div:hover>&]:opacity-100',
+          'opacity-0 transition-opacity [div:hover>&]:opacity-100',
           'flex gap-1 absolute m-2 right-0',
           filename ? 'top-8' : 'top-0'
         ].join(' ')}
       >
-        <Button onClick={toggleWordWrap} className="md:hidden">
+        <Button
+          tabIndex={-1}
+          onClick={toggleWordWrap}
+          className="md:hidden"
+          title="Toggle word wrap"
```

Comment:
same for title and `tabIndex: -1` to avoid focusing on that buttons (because they are invisible) and only visible when user hover code-block